code/modules/mob/mob_helpers.dm

```diff
@@ -531,3 +531,23 @@
 ///Can the mob see reagents inside of containers?
 /mob/proc/can_see_reagents()
 	return stat == DEAD || has_unlimited_silicon_privilege //Dead guys and silicons can always see reagents
+
+/// Sets the mob's hunger levels to a safe overall level. Useful for TRAIT_NOHUNGER species changes.
+/mob/proc/set_safe_hunger_level()
```

Comment:
Could you define this for carbons or humans only? Nutrition is a concept that only makes sense for them.

Reply:
If I'm not completely off base, I think slimes use nutrition.

Reply:
You are right, they do. Slimes are the hackiest simple animals around.
Then `/mob/living/proc/set_safe_hunger_level()`

Reply:
Switched to mob/living/proc and moved to the living.dm file.